File: php/utility/ResultBody.php
<?php
declare(strict_types=1);

// SwissFederalRailwaysSbb SDK utility: result_body

class SwissFederalRailwaysSbbResultBody
{
    public static function call(SwissFederalRailwaysSbbContext $ctx): ?SwissFederalRailwaysSbbResult
    {
        $response = $ctx->response;
        $result = $ctx->result;
        if ($result && $response && $response->json_func && $response->body) {
            $result->body = ($response->json_func)();
        }
        return $result;
    }
}

File: .sdk/tm/rb/utility/result_body.rb
# SwissFederalRailwaysSbb SDK utility: result_body
module SwissFederalRailwaysSbbUtilities
  ResultBody = ->(ctx) {
    response = ctx.response
    result = ctx.result
    if result && response && response.json_func && response.body
      result.body = response.json_func.call
    end
    result
  }
end
